chatgpt-shell-openrouter.el

```diff
@@ -1,6 +1,30 @@
 ;;; chatgpt-shell-openrouter.el --- OpenRouter-specific logic -*- lexical-binding: t; -*-
 
-(cl-defun chatgpt-shell-openrouter-make-model (&rest args &key label version short-version token-width context-window validate-command other-params)
+;; Author: Alvaro Ramirez https://xenodium.com
```

Comment:
Feel free to set or add yourself here. Your call. I'm happy with your choice.

Reply:
Done. Thanks. I wasn't sure if it was meant to be for the file or the project as a whole.